sql/src/test/java/org/apache/druid/sql/calcite/rule/DruidLogicalValuesRuleTest.java

```diff
@@ -101,7 +102,7 @@ private static RexLiteral makeLiteral(Comparable<?> val, SqlTypeName typeName, C
       Mockito.when(dataType.getSqlTypeName()).thenReturn(typeName);
       RexLiteral literal = Mockito.mock(RexLiteral.class);
       try {
-        Field field = literal.getClass().getSuperclass().getDeclaredField("value");
```

Comment:
We can also avoid mocks entirely and make literals directly:

```
    private static RexLiteral makeLiteral(Comparable<?> val, SqlTypeName typeName)
    {
      return (RexLiteral) new RexBuilder(DruidTypeSystem.TYPE_FACTORY).makeLiteral(
          typeName == SqlTypeName.DECIMAL && val != null ? new BigDecimal(String.valueOf(val)) : val,
          DruidTypeSystem.TYPE_FACTORY.createSqlType(typeName),
          false
      );
    }
```

(from https://github.com/apache/druid/issues/14502#issuecomment-1613900607)

Reply:
done